src/System.Net.WebSockets.Client/tests/ConnectTest.cs

```diff
@@ -18,15 +18,18 @@ public class ConnectTest : ClientWebSocketTestBase
 
         [OuterLoop] // TODO: Issue #11345
         [ConditionalTheory(nameof(WebSocketsSupported)), MemberData(nameof(UnavailableWebSocketServers))]
-        public async Task ConnectAsync_NotWebSocketServer_ThrowsWebSocketExceptionWithMessage(Uri server, string exceptionMessage)
+        public async Task ConnectAsync_NotWebSocketServer_ThrowsWebSocketExceptionWithMessage(Uri server, string exceptionMessage, WebSocketError errorCode)
         {
             using (var cws = new ClientWebSocket())
             {
                 var cts = new CancellationTokenSource(TimeOutMilliseconds);
                 WebSocketException ex = await Assert.ThrowsAsync<WebSocketException>(() =>
                     cws.ConnectAsync(server, cts.Token));
 
-                Assert.Equal(WebSocketError.Success, ex.WebSocketErrorCode);
+                if (PlatformDetection.IsNetCore && !PlatformDetection.IsUap) // bug fix in netcoreapp: https://github.com/dotnet/corefx/pull/35960
```

Comment:
Don't the other places in the other tests need to also check `&& !PlatformDetection.IsUap`?

Reply:
I believe they'll all be covered by https://github.com/dotnet/corefx/pull/35960/commits/8fff7e9a9d65ed9cfd600822bf36c1645fc5f636#diff-1878c484b9d09856fac5f31acf13935fR55, but I couldn't run locally to test it, so I'll see what CI says.

Reply:
>I couldn't run locally to test it, so I'll see what CI says.

You'll need to re-do the Outerloop test CI run.  It doesn't happen automatically when you push a new commit.

Reply:
Yup.  I generally just wait until a leg has completed to help validate I didn't mess up the build part.